src/main/scala/djinni/Main.scala

```diff
@@ -235,6 +241,16 @@ object Main {
       opt[String]("py-import-prefix").valueName("<import-prefix>").foreach(pyImportPrefix = _)
         .text("The import prefix used within python generated files (default: \"\")")
       note("\nYaml Generation")
```

Comment:
This line seem to have been copied further below and should probably be deleted.